src/azurelinux/3.0/net10.0/opt/Dockerfile

```diff
@@ -50,3 +48,20 @@ RUN mkdir llvm-project.src && \
         -Wno-dev && \
     make -j $(getconf _NPROCESSORS_ONLN) && \
     make install
+
+
+FROM mcr.microsoft.com/azurelinux/base/core:3.0
+
+RUN tdnf update -y && \
+    tdnf install -y \
+        # Common dependencies
+        ca-certificates \
+        git \
+        tar \
+        util-linux \
+        wget \
+        # Runtime dependencies
+        icu
+
+# Install LLVM that we built from source
+COPY --from=builder /usr/local /usr/local
```

Comment:
Can this be more targeted in what it copies? Because of how Docker works, it will end up copying over all existing files (even though they are the same), essentially duplicating this data between the two image layers. So it's a bunch of wasted space that's caused here.

Reply:
If we set `CMAKE_INSTALL_PREFIX` in the LLVM build command above to another path (can be totally custom), we could copy the `/usr/local` from under that folder and reduce the size hit here.